kuma/users/tests/test_stripe_subscription.py

```diff
@@ -31,9 +31,9 @@ def test_user(db, django_user_model):
     )
 
 
-@patch("kuma.users.utils.create_stripe_customer_and_subscription_for_user")
+@patch("kuma.users.views.create_stripe_customer_and_subscription_for_user")
 @patch(
-    "kuma.users.utils.retrieve_stripe_subscription_info",
+    "kuma.users.views.retrieve_stripe_subscription_info",
```

Comment:
The mock patching isn't of the functions itself. Instead, you're basically monkeypatching the imports in the files that *use* the thing you want to mock.